threeML/plugins/OGIPLike.py

```diff
@@ -718,10 +722,35 @@ def get_simulated_dataset(self, new_name=None):
                 new_ogip_like._mask = original_mask
                 new_ogip_like._apply_mask_to_original_vectors()
 
+            # We want to store the simulated parameters so that the user
+            # can recall them
+
+            new_ogip_like._simulation_storage = clone_model(self._like_model)
+
+            # GV: It is probably safe to assume that there will be only one point source
```

Comment:
I do not understand this comment... where is this assumption made?

Good practice is to add an "assert" clause where this assumption is made, so that the user will be advised if the assertion is violated.